### ISSUE TITLE
[Question] How can i add Bundle-like compatibility for Sacks from the mod Supplementaries?

### ISSUE THREAD

**Issue by JasdanVM** [Kira-NT/packed-inventory#16]
And for colored Sacks from Supplementaries Squared?
As well as tooltip preview, all while having Shulker Boxes only have theire contents visible on the inventory, while having o place them to take them out?

**Comment by Kira-NT** [comment]
> How can i add Bundle-like compatibility for Sacks from the mod Supplementaries? And for colored Sacks from Supplementaries Squared?

Hi! You'll need to implement and register an `InventoryViewer` for each of those items. You can check out [this example](https://github.com/Kir-Antipov/packed-inventory/blob/1.20.x/stable/src/main/java/dev/kir/packedinventory/inventory/PackedInventoryInventoryViewers.java) for guidance.

However, I'd recommend leaving that to the authors of those mods.

> while having Shulker Boxes only have theire contents visible on the inventory, while having o place them to take them out?

Set `validation.minecraft:shulker_box.enable` to `false` in the config.

**Comment by JasdanVM** [comment]
@Kir-Antipov 
Is it not possible to have your mod allow compatibility using Datapacks or some config?

**Comment by Kira-NT** [comment]
Yeah, no. When I first started this mod, the whole data-driven haze hadn't really taken off yet. Additionally, it's not possible to make this mod fully data-driven, as different custom containers provided by other mods often don't adhere to common implementation guidelines, which would still require in-code compatibility adjustments.

However, I do plan to introduce at least some level of data-drivenness to Packed Inventory once I finish refactoring it. This way, at least some common scenarios will only require a JSON file for setup.
